sys/oneway-malloc/oneway-malloc.c

```diff
@@ -7,8 +7,7 @@
  */
 
 /**
- * @addtogroup oneway_malloc
- * @ingroup sys
+ * @ingroup oneway_malloc
```

Comment:
unrelated to sys_memory_management, or should be:

```
 * @defgroup oneway_malloc
 * @ingroup sys_memory_management
 *
```

Reply:
That will cause `oneway_malloc` to be defined twice, I think.

Reply:
right, my bad - thought this is the header, but its not - never mind